### PR TITLE
Json data now displayed properly on the main page using the onload tag

### PR DESCRIPTION
**Pull request by John256145**
The content under /data is now a properly formatted json string. It contains 3 hard-coded comments, and is displayed on the main page using the 'onload' event on the 'body' tag.  (Step 3)

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,45 @@
...
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Juan!");
+    ArrayList<String> jsonarray = new ArrayList<String>();
+    jsonarray.add("I love this site.");
+    jsonarray.add("This is so cool.");
+    jsonarray.add("All the cool kids are coding nowadays.");
+    // response.setContentType("text/html;");
+    // response.getWriter().println("Hello Juan!");
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Can just delete this (to keep the master branch clean)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,45 @@
...
+
+    String json = convertToJson(jsonarray);
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Very admirable you wrote a whole function to do this yourself, but you can just reuse Gson to make things easier :) Check out this example: https://github.com/John256145/software-product-sprint/blob/master/walkthroughs/week-2-server/examples/todo-list/src/main/java/com/google/sps/servlets/ListTasksServlet.java (starting at line 54)

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -16,8 +16,8 @@ <h1>Juan Gonzalez's Portfolio</h1>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
 
-      <p>Click here to recieve my personalized greeting using fetch():</p>
-      <button onclick="addMyGreeting()">Hello, but for Juan</button>
+      <p>Here is some comment data!:</p>
+      <!-- <button onclick="addMyGreeting()">/data</button> -->
       <div id="my-greeting-container"></div>
```
Comment (renguyen) [inline portfolio/src/main/webapp/index.html]:
```suggestion
      <div id="comments-container"></div>
```
nit

**Review by renguyen — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (John256145) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: My portfolio does not use json anymore so I deleted the function instead.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (John256145) [reply portfolio/src/main/webapp/index.html]: This is changed in a later commit and is now "my-comments-container."

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -13,20 +13,38 @@
...
-    response.getWriter().println("Hello Juan!");
+    ArrayList<Comment> jsonarray = new ArrayList<Comment>();
+    Comment newcomment1 = new Comment("I love this site.");
+    Comment newcomment2 = new Comment("This is so cool.");
+    Comment newcomment3 = new Comment("All the cool kids are coding nowadays.");
+    jsonarray.add(newcomment1);
+    jsonarray.add(newcomment2);
+    jsonarray.add(newcomment3);
+
+    String json = convertToJson(jsonarray);
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]:
```suggestion
    ArrayList<Comment> jsonArray = new ArrayList<Comment>();
    Comment newComment1 = new Comment("I love this site.");
    Comment newComment2 = new Comment("This is so cool.");
    Comment newComment3 = new Comment("All the cool kids are coding nowadays.");
    jsonArray.add(newComment1);
    jsonArray.add(newComment2);
    jsonArray.add(newComment3);

    String json = convertToJson(jsonArray);
```

nit: variable names should be in lowerCamelCase in Java

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -13,20 +13,38 @@
...
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+
+  }
+
+  private String convertToJson(ArrayList<Comment> alist) {
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: rename alist to comments/commentsList to make it clear what the variable contains

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="addMyComments()">
```
Comment (renguyen) [inline portfolio/src/main/webapp/index.html]: Doesn't have to be done in this PR but does it still work if you move `onload="addMyComments()"` to `<div id="comments-container">`?

Comment (John256145) [reply portfolio/src/main/webapp/index.html]: Yes, 
I moved it to line 20: 
`<script>addMyComments();</script>`

**Review by renguyen — APPROVED** [review]
Thank you for making all the changes! :)

**Pull request merged**